src/Config/ObjectModel/AuthProvider.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Config.ObjectModel;
+
+public enum AuthProvider
+{
+    StaticWebApps,
```

Comment:
Do we have plans to remove `EasyAuthType` Enum?

Reply:
Not yet, since its being used in the code to ValidateAudienceAndIssuerForJwtProvider

Reply:
can you create an issue to refactor, this can be put in backlog?